_component_design/date-picker.md

```diff
@@ -0,0 +1,341 @@
+---
+title: Date Picker
+parent: Inputs and Controls
+layout: component
+category: Components
+usage: Enable users to filter a list of events, or a report, by date ranges, or choose a single date for an event as they fill out a form.
+preview-image: preview-images/date-picker.svg
+status: unknown
+resource: true
+last-modified: 2017-10-22
+---
+
+{% include toc.html %}
+
+## Introduction
+
+<div class="hxRow">
+{% column left:"hxCol-4 hxCol-xs-12 hxCol-sm-12 hxCol-md-4 hxCol-lg-4" %}
+
+### When to use
+Date picker is often used for applying date range filters to lists of events, logs, or even for filtering monitoring reports.  It can also be used for advanced filtering in areas such as ticketing and billing.  The picker may be used to specify dates or ranges in the process of submitting a form for use cases other than filtering as well.
```

Comment:
Trying to simplify this a bit:
"You can use the date picker to apply date range filters to lists of events, logs, or for filtering monitoring reports. You can also use it for advanced filtering in ticketing and billing by specifying dates or date ranges in the process of submitting a form."

Reply:
excellent guidance.  thank you!